hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/TestUpdateSchemaEvolution.java

```diff
@@ -68,61 +71,62 @@ public void tearDown() throws IOException {
     cleanupResources();
   }
 
-  @Test
-  public void testSchemaEvolutionOnUpdate() throws Exception {
+  private WriteStatus prepareFirstCommitData(List<String> recordsStrs) throws IOException {
```

Comment:
to avoid ambiguity, could we call this `prepareFirstRecordCommit()`?

Reply:
thanks, make sense